### DISCUSSION TITLE
Default container image 2.8.2 switched from Python 3.8 to 3.11

### DISCUSSION THREAD

**Discussion by sseelmann** [apache/airflow#37882] [General]
Hello,

I noticed that the default container images for version 2.8.2 now uses Python 3.11, whereas the one from version 2.8.1 uses Python 3.8. I just wonder if that was a conscious change as it's a bugfix release.  

https://hub.docker.com/layers/apache/airflow/2.8.2/images/sha256-a9a409c0474e0266045b7d300875af5231d87f1fabd1ad45d4fe246f5879acc0?context=explore

https://hub.docker.com/layers/apache/airflow/2.8.1/images/sha256-0f1a44b37f13ea968066cac532cd488f4736b942bd83dde15483cb65a9cafe17?context=explore

Thanks,
Stefan

**Comment by Taragolis** [comment]
[Docker Images](https://airflow.apache.org/docs/docker-stack/changelog.html) do not follow strict SemVer, the original changes should happen in Airflow 2.8, see [Discussion](https://lists.apache.org/thread/xy192wmv6k1sy2wlx5vy64p678skrhko) and [Lazy Consensus](https://lists.apache.org/list?dev@airflow.apache.org:2023-11:Docker) and clarification of [Lazy Consensus](https://lists.apache.org/list?dev@airflow.apache.org:2023-12:Docker) but I guess it originally postponed to Airflow 2.9 images.
However, who knows maybe one of the recent rebuild brings this changes a bit earlier than expected.

And some my personal thoughts, use non-python specific tags bring this changes sooner or later, I would recommend always use `-pythonX.Y` rather than default one.

**Comment by potiuk** [comment]
I reverted it back. the 2.8.2 default image is back 3.8 (and we will only change default for 2.9

**Reply by potiuk** [reply]
It was indeed a fix brought too early
